.nuspec/Xamarin.Forms.Maps.nuspec

```diff
@@ -18,12 +18,14 @@
         <dependency id="Xamarin.Forms" version="$version$"/>
       </group>
       <group targetFramework="MonoAndroid10.0">
-        <dependency id="Xamarin.AndroidX.Migration" version="1.0.0-rc1"/>
-        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.1.0-rc1"/>
-        <dependency id="Xamarin.AndroidX.Legacy.Support.Core.UI" version="1.0.0-rc1"/>
-        <dependency id="Xamarin.AndroidX.AppCompat" version="1.1.0-rc1"/>
+        <dependency id="Xamarin.AndroidX.Migration" version="1.0.0"/>
+        <dependency id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.1.0"/>
+        <dependency id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0"/>
+        <dependency id="Xamarin.AndroidX.Legacy.Support.Core.UI" version="1.0.0"/>
+        <dependency id="Xamarin.AndroidX.AppCompat" version="1.1.0"/>
         <dependency id="Xamarin.GooglePlayServices.Maps" version="71.1610.0"/>
         <dependency id="Xamarin.Forms" version="$version$"/>
+        <dependency id="Xamarin.Android.Support.v4" version="28.0.0.3"/>
```

Comment:
why do we need this dependency?

Reply:
Because of this dependency chain

Xamarin.GooglePlayServices.Maps 71.1610.0 -> Xamarin.GooglePlayServices.Basement 71.1620.0 -> Xamarin.Android.Support.v4 28.0.0.1 -> Xamarin.Android.Support.Core.UI (= 28.0.0.1).

which leads to these exceptions
```
Version conflict detected for Xamarin.Android.Support.Core.UI. Install/reference Xamarin.Android.Support.Core.UI 28.0.0.3 directly to project Xappy.Android to resolve this issue. 
 Xappy.Android -> Com.Airbnb.Xamarin.Forms.Lottie 3.0.4 -> Xamarin.Android.Support.v7.AppCompat 28.0.0.3 -> Xamarin.Android.Support.Fragment 28.0.0.3 -> Xamarin.Android.Support.Core.UI (= 28.0.0.3) 
 Xappy.Android -> Xamarin.Forms.Maps 4.6.0.314-nightly -> Xamarin.GooglePlayServices.Maps 71.1610.0 -> Xamarin.GooglePlayServices.Basement 71.1620.0 -> Xamarin.Android.Support.v4 28.0.0.1 -> Xamarin.Android.Support.Core.UI (= 28.0.0.1).
Restore failed.
```

And the only way users can fix the exception is to add `<dependency id="Xamarin.Android.Support.v4" version="28.0.0.3"/>` themselves so we're just adding it as a dependency for now to make it easier


Once there is a Xamarin.GooglePlayServices.Maps 71.1610.0 which is all androidx or depends on 28.0.0.3 then we can remove